Dockerfile

```diff
@@ -1,19 +1,15 @@
 FROM node:0.10-slim
 MAINTAINER matthias.luebken@gmail.com
-WORKDIR /app
 
-# install dependencies
-ADD package.json /app/
-RUN npm install
+LABEL Version="1.0.0"
```

Comment:
I like it. Do you have more information on this convention. E.g. how this relates to the tag?